pages/AdListPage/[id]/index.js

```diff
@@ -0,0 +1,122 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import { fetcher } from "../../../helpers/api";
+import styled from "styled-components";
+
+export default function AdDetailsPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const { data: ad, error } = useSWR(`/api/ads/${id}`, fetcher);
+
+  if (error) return <h1>...sorry cannot load ads</h1>;
+
+  if (!ad) return <h1>...please wait while loading...</h1>;
+  console.log(ad);
+
+  return (
+    <StyledArticle>
+      <UserContainer>
+        <UserProfilPhoto
+          src={
+            ad.userPictureSrc == ""
+              ? `https://source.unsplash.com/random/?person ${ad.userName}`
+              : ad.userPictureSrc
+          }
+          width={40}
+          height={40}
+          alt={`Profilphoto of ${ad.userName}`}
+        ></UserProfilPhoto>
+        <h2>{ad.userName}´s Ad</h2>
+      </UserContainer>{" "}
```

Comment:
-  next/image is self closing therefor the closing tag aren't needed  
 - I recommend to set a default image instead of a random image from unsplash.com - by the way the path looks some kind of strange:  a whitespace but no `/` does that really work? 🤨 

```suggestion
        <UserProfilPhoto
          src={
            ad.userPictureSrc == ""
              ? `https://source.unsplash.com/random/?person ${ad.userName}`
              : ad.userPictureSrc
          }
          width={40}
          height={40}
          alt={`Profilphoto of ${ad.userName}`}
        />
        <h2>{ad.userName}´s Ad</h2>
      </UserContainer>
```